Examples/Python/src/Geant4Component.cpp

```diff
@@ -69,7 +69,14 @@ PYBIND11_MODULE(ActsPythonBindingsGeant4, mod) {
              std::shared_ptr<DetectorConstructionFactory>>(
       mod, "DetectorConstructionFactory");
 
-  py::class_<Geant4Handle, std::shared_ptr<Geant4Handle>>(mod, "Geant4Handle");
+  py::class_<Geant4Manager, std::unique_ptr<Geant4Manager, py::nodelete>>(
```

Comment:
Unique nodelete means that Python will not own this object. Does G4 own it?

Reply:
`Geant4Manager` is a singleton and has a private destructor so python should really not own it. without this it also complains about not being able to correctly deconstruct it

Reply:
Huh, I think I didn't appreciate that this was an option. I worked around this using factory functions in some cases, but this might be a bit nicer.